_posts/2018-07-17-its-a-lot-more-snappy.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: post
+title: "It's A Lot More Snappy"
+date: 2018-07-17
+categories: PWA SPA JavaScript
+author: Rick Foxcroft
+image: img/repeatable-serverless-applications/less_servers.png
+---
+
+**It's A Lot More Snappy**
```

Comment:
This is the only heading in the post, and it's quite long.

Again, reviewing markdown only so don't have the benefit of overall look, but would suggest adding a couple of headings to help break up the flow - a heading provides a mental reset point for the reader, giving chance for you to re-engage them if they were starting to drift.

Reply:
Actually - it's not a heading.  Should it be?

Reply:
Also quotation marks - it's a quote.